lib/ftp.c

```diff
@@ -1499,22 +1497,16 @@ static CURLcode ftp_state_list(struct connectdata *conn)
 
 static CURLcode ftp_state_retr_prequote(struct connectdata *conn)
 {
-  CURLcode result = CURLE_OK;
-
   /* We've sent the TYPE, now we must send the list of prequote strings */
-
-  result = ftp_state_quote(conn, TRUE, FTP_RETR_PREQUOTE);
+  CURLcode result = ftp_state_quote(conn, TRUE, FTP_RETR_PREQUOTE);
```

Comment:
This could even do a `return` straight away without storing the return code in a variable...